kms/tpmkms/tpmkms.go

```diff
@@ -304,39 +326,76 @@ func (k *TPMKMS) CreateKey(req *apiv1.CreateKeyRequest) (*apiv1.CreateKeyRespons
 }
 
 // CreateSigner creates a signer using a key present in the TPM KMS.
+//
+// The `signingKey` in the [apiv1.CreateSignerRequest] can be used to specify
+// some key properties. These are as follows:
+//
+//   - name=<name>: specify the name to identify the key with
+//   - path=<file>: specify the TSS2 PEM file to use
```

Comment:
I'll think about this some more w.r.t. what it means for the `tpm.TPM` storage implementation, because there are files there too. I think adding `path` is OK as long as it's clear that those are for keys _outside_ of the TPM storage directory (usually). I would also like to keep the option open for a "native TSS2 storage".

Reply:
This is similar to the `path` used in `softkms`, where you can reference a file using `softkms:path=/dir/to/file.pem`. So yes, it's a file outside the TPM storage.

Yes, it also makes sense to create a native TSS2 storage, but we need to think a little bit more about it, because we will need to store certificates too. But once is done I imagine it will be similar to the current one and we will be using `name`.

Reply:
The simplest approach might be to have a separate file (with the same or a derived name) with the cert next to the TSS2 PEM key. And possibly also a file with additional metadata, which could hold creation / attestation data.